benchmarks/eval_vlm_vision.py

```
@@ -0,0 +1,203 @@
+        return float(m.group(1)) / 1.073741824
+    return None
+
+
+def run(model, max_tokens=128):
+    build_images()
+    module, temp_flag = engine_for(model)
+    print(f"model: {model}   (via {module})\n")
+    cases, peaks = [], []
+    for name, prompt, accept in CASES:
+        t0 = time.time()
+        proc = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)